docs/migration-guides/classic-iqp-to-cloud-iqp.mdx

```diff
@@ -142,11 +143,11 @@ Follow these steps to start using the new version of IBM Quantum Platform.
     service = QiskitRuntimeService(name="account-name")
     ```
 
-      * If you have saved credentials for both an IBM Quantum Platform account and an IBM Cloud account, IBM Cloud is the default account used when you initialize your account, unless you specify `set_as_default=True` in your IBM Quantum Platform account when you use the `save_account()` method.
-      * If you previously saved an IBM Quantum Platform account as the default, specify `set_as_default=True` for your new cloud account when you use the `save_account()` method.
+      * If you previously saved an IBM Quantum Platform classic account as the default, specify `set_as_default=True` for your new cloud account when you use the `save_account()` method.
+      * If no accounts are saved as the default, an account saved on the new IBM Quantum Platform with `ibm_quantum_platform` as the channel name will be used as the default when initializing your account. `set_as_default=True` can still be used to set an account with a certain configuration of credentials as the default.
       * If you are saving multiple accounts per channel, consider using the `name` parameter to differentiate them.
       * Credentials are saved to `$HOME/.qiskit/qiskit-ibm.json`.  Do not manually edit this file.
-1. <span id="untrusted-machine"></span>**Avoid executing code on an untrusted machine or an external cloud Python environment to minimize security risks.** If you must use an untrusted environment (for example, a public computer), change your API key after each use by following the instructions in the [Delete an API key](https://cloud.ibm.com/docs/account?topic=account-userapikey&interface=ui#delete_user_key) section to reduce risk. To initialize the service in this situation, you can use code like the following:
+5. <span id="untrusted-machine"></span>**Avoid executing code on an untrusted machine or an external cloud Python environment to minimize security risks.** If you must use an untrusted environment (for example, a public computer), change your API key after each use by following the instructions in the [Delete an API key](https://cloud.ibm.com/docs/account?topic=account-userapikey&interface=ui#delete_user_key) section to reduce risk. To initialize the service in this situation, you can use code like the following:
```

Comment:
Just an FYI (no need to change) but when enumerating in mdx, we can use repeated `1`s because they render as an enumerated list, and then future rearranging/adding/removing items in a numbered list is easy.